WordPress/src/main/java/org/wordpress/android/util/SnackbarSequencer.kt

```diff
@@ -0,0 +1,114 @@
+package org.wordpress.android.util
+
+import android.app.Activity
+import android.content.Context
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.modules.UI_THREAD
+import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.AppLog.T
+import org.wordpress.android.widgets.WPSnackbar
+import org.wordpress.android.widgets.WPSnackbarWrapper
+import java.util.LinkedList
+import java.util.Queue
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.coroutines.CoroutineContext
+
+const val DURATION_EXTRA_MARGIN = 500L
```

Comment:
I think we should increase the MARGIN_DURATION, since it's currently causing this little animation twitch.
![snackbar-short-margin](https://user-images.githubusercontent.com/2261188/69712548-552fee00-1103-11ea-8793-fb2b37f74b86.gif)
Wdyt?

Reply:
@develric I just found out that the system handles the animation automatically when you set the MARGIN_DURATION to 0L. In that case I think we should completely remove it. Wdyt?

Reply:
I'm not sure to fully understand the why of this behavior, but that's it 🤷‍♂️! Removed.